pandas/tests/groupby/test_groupby.py

```diff
@@ -32,6 +32,20 @@
 from pandas.core.groupby.base import maybe_normalize_deprecated_kernels
 
 
+def test_group_by_copy():
```

Comment:
hmm actually should be in `pandas/tests/groupby/test_apply_mutate.py`

@NumberPiOso if you wouldn't mind doing a PR to move this.

Reply:
Yes! ,  PR#45578